examples/datagen_with_al/README.md

```diff
@@ -0,0 +1,221 @@
+# Training with Active Learning
```

Comment:
Modify "Training with Active Learning" -> "Training with Active Learning (AL)" as AL is used as abbreviation in next part of the readme.

Reply:
Thanks, I will make sure to use abbreviations consistently!